test/LifToken.js

```diff
@@ -132,6 +132,83 @@ contract('LifToken', function(accounts) {
     await help.checkToken(token, accounts, 125, [40,30,20,10,0]);
   });
 
+  it('should fail when calling a function that fails inside approveData', async function() {
```

Comment:
I think it would be nice to explain how it fails here, like `should fail and not trigger an ApproveData event...`